pdns/pdnsutil.cc

```diff
@@ -604,15 +604,41 @@ int checkAllZones(DNSSECKeeper &dk, bool exitOnError)
 {
   UeberBackend B("default");
   vector<DomainInfo> domainInfo;
+  struct name{};
+  struct id{};
```

Comment:
I'm probably missing something but I don't see why you declare these?

Reply:
You're not missing anything, they were leftover tag structs, but then I got rid of their usage!